rethinkdb/tests/test_integration.py

```diff
@@ -26,116 +24,97 @@
 )
 from .types import ServerName
 
-try:
-    from contextlib import nullcontext  # type: ignore
-except ImportError:
-    from contextlib2 import nullcontext
-
 
 @pytest.mark.integration
 @pytest.mark.usefixtures('dd_environment')
 class TestCheck:
     METRICS_COLLECTION_MOCK_TARGET = 'datadog_checks.rethinkdb.check.RethinkDBCheck.collect_metrics'
 
-    def run_test(
-        self, aggregator, instance, check_context=None, connect_status=RethinkDBCheck.OK, disconnected_servers=None
-    ):
-        # type: (AggregatorStub, Instance, ContextManager[None], ServiceCheckStatus, Set[ServerName]) -> None
+    def test_default(self, dd_run_check, aggregator, instance):
```

Comment:
I rearranged these tests to just be very explicit and drop the `.run_test()` helper; it wasn't useful anymore because of `dd_run_check` which changed how the check was run anyway.